tests/model_explainability/guardrails/conftest.py

```
@@ -133,3 +153,436 @@ def hap_detector_route(
         service=hap_detector_isvc.name,
         wait_for_resource=True,
     )
+
+
+@pytest.fixture(scope="class")
+def installed_tempo_operator(admin_client: DynamicClient, model_namespace: Namespace) -> Generator[None, Any, None]:
```

Ruff (0.14.1)

Unused function argument: `model_namespace`

(ARG001)